components/automate-backend-deployment/migration/A2HA-to-AutomateHA/migration.md

```diff
@@ -0,0 +1,60 @@
+In some scenario we are required to migrate A2HA data to Automate HA cluster`(as we have new HA implementation in Automate)`. For that here are some steps that you'll have to follow.
+
+
+1) Take backup of chef-automate from A2HA `(Old)` using below command, this command can be executed from any of front-end (chef-automate) node  
+ in case of multiple frontends. 
+ Usually if you don't specify any location for backup in config.toml then that backup will be store on /var/opt/chef-automate/backup location 
+ if you hit below command.
+
+    `sudo chef-automate backup create`
+    
+
+2) Make a tar file of backup that we have taken in step 1. Here make sure that you are also taking backup of `.tar` directory. Otherwise you'll face some issue related to metadata.json.
+
+    E.g. `tar -cvf backup.tar.gz path/to/backup/20201030172917/ /path/to/backup/automate-backup-elasticsearch/ /path/to/backup/.tmp/`
+   
+
+
+3) Create a aib file from any of chef-automate frontend node of A2HA. This will create a bundle of all necessary keys. Like pivotal.pem, secret key etc. Usually this will not be included in regular backup`(step 1)` so make sure you create a bundle for that.
```

Comment:
```suggestion
3) Create a abb file from any of chef-automate frontend node of A2HA. This will create a bundle of all necessary keys. Like pivotal.pem, secret key etc. Usually this will not be included in regular backup`(step 1)` so make sure you create a bundle for that.
```